android/src/main/kotlin/eu/nk2/tinkoff_acquiring_sdk/TinkoffAcquiringSdkDelegate.kt

```diff
@@ -37,16 +38,15 @@ class TinkoffAcquiringSdkDelegate(private val activityDelegate: ActivityDelegate
         if(tinkoffAcquiring != null || googlePayHelper != null) return TinkoffAcquiringDelegateInitializeResponse(status = TinkoffAcquiringDelegateInitializeStatus.PLUGIN_ALREADY_INITIALIZED)
 
         tinkoffAcquiring = TinkoffAcquiring(terminalKey, password, publicKey)
-        if(enableDebug) {
-            AcquiringSdk.isDeveloperMode = true
-            AcquiringSdk.isDebug = true
-        }
+      
+        AcquiringSdk.isDeveloperMode = isDeveloperMode
+        AcquiringSdk.isDebug = isDebug
 
         return if(!enableGooglePay) TinkoffAcquiringDelegateInitializeResponse(status = TinkoffAcquiringDelegateInitializeStatus.RESULT_OK)
         else {
             googlePayHelper = GooglePayHelper(GooglePayParams(
                 terminalKey = terminalKey,
-                environment = if (enableDebug) WalletConstants.ENVIRONMENT_TEST else WalletConstants.ENVIRONMENT_PRODUCTION,
+                environment = if (isDeveloperMode) WalletConstants.ENVIRONMENT_TEST else WalletConstants.ENVIRONMENT_PRODUCTION,
```

Comment:
Maybe its better to use another naming for this flag (smth. like `useDeveloperCredentials`)